infrastructure/base/modules/ecr/main.tf

```diff
@@ -0,0 +1,69 @@
+##
+# Module to build the ECR repository
+##
+
+# TODO: must be a cleaner way to do it than to duplicate the resource
```

Comment:
yeah, you could, but it's not really worth it

Reply:
:)